components/compliance-service/inspec/cli.go

```diff
@@ -264,33 +320,94 @@ func Check(profilePath string) (CheckResult, error) {
 		return res, errors.New(strings.Join(errs, "\n"))
 	}
 
-	logrus.Debugf("Successfully checked inspec profile in %s", profilePath)
+	logrus.Infof("Successfully checked inspec profile in %s", profilePath)
 	return res, nil
 }
 
-func Json(profilePath string) ([]byte, error) {
-	args := []string{shimBinName, "json", profilePath}
-	logrus.Debugf("Run: inspec %v", args)
-	stdout, stderr, err := run(args, nil, defaultTimeout, inspecShimEnv())
-	logrus.Debugf("Run: %s %s %v", stdout, stderr, err)
+func Json(profilePath string, firejailprofilePath string) ([]byte, error) {
```

Comment:
can we combine the logic of Json and Check function.

Reply:
Nopes, these are two commands which needed to be run parlely. Both doing the different operations. This was like this before, we just added few things